### PR TITLE
DN4J-10: Support neo4j v5 in docker-compose stack

### PR DESCRIPTION
**Pull request by thibveni**

### CODE REVIEW

**Comment by thibveni** [comment]
i retested both v4 & v5. works fine

File: .env.neo4j.example [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+NEO4J_AUTH=neo4j/neo3jneo3j
 
 # test restore
-DATASETS=aml-1.0.0
-NEXUS_TOKEN=
-DATASET_NEO4J_VERSION=4.2.0
-
+# DATASETS=aml-1.0.0
+# NEXUS_TOKEN=
+# DATASET_NEO4J_VERSION=4.2.0
```
Comment (enys) [inline .env.neo4j.example:11]: not sure this needs to be commented as it's an example

**Review by enys — APPROVED** [review]
Perhaps mention the new v5 stack in the Readme ?

File: .env.neo4j.example [hunk 2]
```diff
@@ -1,11 +1,11 @@
-NEO4J_dbms_memory_heap_initial__size=512M
-NEO4J_dbms_memory_heap_max__size=512M
-NEO4J_dbms_memory_pagecache_size=512M
+NEO4J_server_memory_heap_initial__size=512M
+NEO4J_server_memory_heap_max__size=512M
+NEO4J_server_memory_pagecache_size=512M
+
```
Comment (enys) [inline .env.neo4j.example:4]: so actualy this will only work in v5

Comment (thibveni) [reply .env.neo4j.example:4]: oops, v4 was started but yep these variables were not taken in account, redoing a pr

**Pull request merged**